services/php-version.js

```diff
@@ -2,14 +2,23 @@
  * Utilities relating to PHP version numbers. This compares version numbers
  * using the algorithm followed by Composer (see
  * https://getcomposer.org/doc/04-schema.md#version).
+ *
+ * @module
  */
+
 import { fetch } from '../core/base-service/got.js'
 import { getCachedResource } from '../core/base-service/resource-cache.js'
 import { listCompare } from './version.js'
 import { omitv } from './text-formatters.js'
 
-// Return a negative value if v1 < v2,
-// zero if v1 = v2, a positive value otherwise.
+/**
+ * Return a negative value if v1 < v2,
+ * zero if v1 = v2, a positive value otherwise.
+ *
+ * @param {number} v1 - First version for comparison
+ * @param {number} v2 - Second version for comparison
```

Comment:
I think `v1` and `v2` are actually strings (they can be something like "1.0.0", for example)

Reply:
You are right. I've updated the PR now. Thanks